doc/nrf/documentation/glossary.rst

```diff
@@ -91,6 +91,11 @@ Glossary
       The different sources can include source files such as :file:`main.c` and configuration input files such as :ref:`Devicetree <configure_application_hw>` and :ref:`Kconfig <configure_application_sw>` files.
       The build scripts are then used during the :ref:`configuration_system_overview_build` to create the application firmware.
 
+   Build type
+      A build type is a CMake feature that defines the way in which the configuration files are to be handled.
```

Comment:
The only thing I has a problem with is that zephyr's build type does not have anything to do with cmake_build_type.
This sentence can cause confusion as some people may think that either setting this variable will have an effect or that Zephyr will make something with this variable when specifying suffixed config file.

Reply:
Removed references to CMake: "A build type is a feature that..."